show/bgp_frr_v4.py

```diff
@@ -35,7 +35,7 @@ def summary(namespace, display):
 def neighbors(ipaddress, info_type):
     """Show IP (IPv4) BGP neighbors"""
 
-    command = 'sudo vtysh -c "show ip bgp neighbor'
+    command = 'sudo rvtysh -c "show ip bgp neighbor'
```

Comment:
>rvtysh [](http://example.com/codeflow?start=20&length=6)

Could you use the new constant `RVTYSH_COMMAND`? Or follow my other comment to define a new function `run_bgp_show_command` so you don't need to prefix with `sudo rvtysh -c`

Reply:
@qiluo-msft , do you mean to change all the show commands to use the the new function?

Reply:
yes

Reply:
@qiluo-msft , could you please take a look again?